example/contacts/contacts.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+package api.contacts;
+
+import "google/protobuf/empty.proto";
+import "google/api/annotations.proto";
+import "github.com/lyft/protoc-gen-validate/validate/validate.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+import "github.com/infobloxopen/protoc-gen-gorm/options/orm.proto";
+
+option go_package = "github.com/infobloxopen/protoc-gen-gorm/example/contacts;contacts";
+
+message Contact {
+    option (orm.opts).ormable = true;
+    uint64 id = 1;
+    string first_name = 2;
+    string middle_name = 3;
+    string last_name = 4;
+    string email_address = 5 [(validate.rules).string.email = true];
+}
+
+// eventually this will be replaced with a Page that uses the paging info
+message ContactPage {
+    repeated Contact results = 1;
+}
+
+// eventually this will be replaced with a standard search request
+message SearchRequest {
+    string first_name = 1;
+}
+
+message GetRequest {
+    uint64 id = 1;
+}
+
+message SMSRequest {
+    uint64 id = 1;
+    string message = 2;
+}
+
+service Contacts {
```

Comment:
This is ok for now, I think though that we want to nail down the conventions. @dmacthedestroyer suggested CreateRequest, for example. So, ultimately this example should follow those conventions (or we should just have the one example app and not put all the examples in here at all).

Something like:

```
message Contact {
...
}

message CreateRequest {
  Contact contact = 1;
  // maybe some other control or metadata, for example, a certain app may want to add
  // options to the create
  bool overwrite = 2; // overwrite an existing contact with the same email address
  // or add tags
  TagSet tags = 3;
}

message CreateResponse {
  Contact contact = 1;
}

service Contacts {
...
```

We need to have a separate discussion about what we want those basic calls to look like. They should be built in a way that they are extensible and integrate with the other Atlas services.